src/utils/get-current-player-height.js

```diff
@@ -4,8 +4,8 @@ export const getCurrentPlayerHeight = (function() {
   return function getCurrentPlayerHeight(world) {
     avatarPOV = avatarPOV || document.getElementById("avatar-pov-node");
     avatarRig = avatarRig || document.getElementById("avatar-rig");
-    avatarPOV.object3D.updateMatrices();
     avatarRig.object3D.updateMatrices();
+    avatarPOV.object3D.updateMatrices();
```

Comment:
This reordering is unrelated to the bug, but it just seems strange to me to update child then parent instead of parent then child.